optuna_dashboard/ts/components/GraphEdf.tsx

```diff
@@ -29,7 +29,7 @@ const GraphEdfBackend: FC<{
   const { graphComponentState, notifyGraphDidRender } = useGraphComponentState()
 
   const studyIds = studies.map((s) => s.id)
-  const domId = getPlotDomId(-1)
+  const domId = getPlotEdfDomId(-1)
```

Comment:
I think that using the same `id` attribute for `GraphEdfBackend` and `PlotEdf` component isn't necessary. How about we consider removing the `getPlotEdfDomId` from `@optuna/react`?